src/views/shelter/killShelters.handlebars

```diff
@@ -0,0 +1,18 @@
+<h1>{{title}}</h1>
+
+{{#if shelters.length}}
+    <ul>
+        {{#each shelters}}
+            <li class="tabClass" ><a href="/shelters/{{this._id}}">{{this.shelterName}}</a></li>
+            <li>Pets at this shelter have at most {{this.timeKept}} days until they are euthanized</li>
```

Comment:
Have all the shelter Information for each one inside a nested <ul> tag. Like this:
`    <ul>
        {{#each shelters}}
            <li class="tabClass" ><a href="/shelters/{{this._id}}">{{this.shelterName}}</a></li>
            <ul>
                <li>Pets at this shelter have at most {{this.timeKept}} days until they are euthanized</li>
                <li>There are currently {{this.numberPets}} pets at this shelter</li>
                {{#if this.pets.length}}  
                    <ul>
                    {{#each this.pets}}
                        <li>{{this.name}}</li>
                    {{/each}}
                    </ul>  
                {{/if}}
            </ul>
        {{/each}}
    </ul>`

Reply:
That looks much better! Thanks!

Reply:
<ul> 
    {{#each shelters}} 
        <li class="tabClass" ><a href="/shelters/{{this._id}}">{{this.shelterName}}</a> 
        <ul> 
            <li>Pets at this shelter have at most {{this.timeKept}} days until they are euthanized</li> 
            <li>There are currently {{this.numberPets}} pets at this shelter
            {{#if this.pets.length}} 
                <ul> 
                    {{#each this.pets}} 
                        <li>{{this.name}}</li> 
                    {{/each}} 
                </ul> 
            {{/if}} 
            </li>
        </ul> 
        </li>
    {{/each}}
</ul>